nss-day-cohort-39/tabloid-mvc-depech-mode
# Comments on Post

### What I did:
- Created a Comment Controller for listing, editing, deleting, creating new comments
- Made a Comment Repository to set up the logic for these
- Styled to our standards, I think?
- Created view for comment (Index, Detail, Edit and Delete for comments)
- Added the list of comments to the detail view for Post
### How to test:
- Click on a Post and My Post 
- Click on Detail view
- Click the View Comments Button, which is at bottom below a picture if you have one
- The comment list should have a subject, content, creation date, and author display name on it
- Test out editing and deleting comments (should only be able to edit content and subject)
- Test out Adding a comment, which you have to click on the Add comment button
- Make sure cancel directs you back to your list of comments for each of these
